assignments/array-methods.js

```diff
@@ -55,14 +55,31 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
-let fullName = [];
-console.log(fullName);
+let fullNames = [];
+
+function getFullnames(arr) {
+arr = fullNames;
+
+    for(let i = 0; i < 50; i++) {
```

Comment:
instead of using i < 50; youcould do
```suggestion
    for(let i = 0; i < arr.length; i++) {
    
    }
```